lighthouse-core/test/lib/icons-test.js

```diff
@@ -328,4 +329,73 @@ describe('Icons helper', () => {
       assert.equal(icons.pngSizedAtLeast(192, manifest.value).length, 1);
     });
   });
+
+  describe('icons at least one maskable check', () => {
+    it('succeeds when at least one icon has a purpose value of maskable', () => {
+      const manifestSrc = JSON.stringify({
+        icons: [{
+          src: 'icon.png',
+          sizes: '200x200',
+          type: 'image/png',
+          purpose: 'any',
+        }, {
+          src: 'icon-vector.svg',
+          sizes: '100x100',
+          purpose: 'maskable',
+        }],
+      });
+      const manifest = manifestParser(manifestSrc, EXAMPLE_MANIFEST_URL, EXAMPLE_DOC_URL);
+      assert.equal(icons.containsMaskableIcon(manifest.value), true);
+    });
+
+    it('succeeds when multiple icons have a purpose value of maskable', () => {
+      const manifestSrc = JSON.stringify({
+        icons: [{
+          src: 'icon.png',
+          sizes: '200x200',
+          type: 'image/png',
+          purpose: 'maskable',
+        }, {
+          src: 'icon-vector.svg',
+          sizes: '100x100',
+          purpose: 'maskable',
+        }],
+      });
+      const manifest = manifestParser(manifestSrc, EXAMPLE_MANIFEST_URL, EXAMPLE_DOC_URL);
+      assert.equal(icons.containsMaskableIcon(manifest.value), true);
+    });
+
+    it('succeeds when an icon has multiple purpose values, including maskable', () => {
+      const manifestSrc = JSON.stringify({
+        icons: [{
+          src: 'icon.png',
+          sizes: '200x200',
+          type: 'image/png',
+          purpose: 'any maskable',
```

Comment:
```suggestion
          purpose: 'any Maskable',
```

just to check our lowercase-ness maybe? :)